definitions/ext-factorio_game/dashboard.json

```diff
@@ -0,0 +1,520 @@
+{
+  "name": "Factorio Dashboard",
+  "pages": [
+    {
+      "name": "Factorio Dashboard",
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 1,
+            "width": 1
+          },
+          "title": "",
+          "rawConfiguration": {
+            "dataFormatters": [
+              {
+                "name": "hours played",
+                "precision": 1,
+                "type": "decimal"
+              }
+            ],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT latest(factorio_tick) / 100000 as 'hours played'"
+              }
+            ],
+            "thresholds": []
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 2,
+            "row": 1,
+            "height": 1,
+            "width": 1
+          },
+          "title": "",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT latest(factorio_connected_players) as 'connected players'"
+              }
+            ],
+            "thresholds": []
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 3,
+            "row": 1,
+            "height": 1,
+            "width": 1
+          },
+          "title": "",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT latest(factorio_connected_players) as 'total players'"
```

Comment:
how is this different from connected players?

Reply:
From [Factorio's API Docs](https://lua-api.factorio.com/latest/index.html)
- [Players](https://lua-api.factorio.com/latest/LuaGameScript.html#LuaGameScript.players): Get a table of all the players that currently exist.
- [Connected players](https://lua-api.factorio.com/latest/LuaGameScript.html#LuaGameScript.connected_players): The players that are currently online.
In an online server existing players may not be connected.

Reply:
oooh 🤦‍♂️